From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Mon, 1 Mar 2021 23:11:06 -0800
Subject: [PATCH 19/23] soong: Disable clang-tidy

Running clang-tidy on all the AOSP code is of little use for us, but it
takes a substantial amount of build time. Disable it to reduce build
times for native code.

Change-Id: Idc3d97bf0a08db31e66ab6ad018749575906ca1e
Signed-off-by: Anushek Prasal <anushekprasal@gmail.com>
Signed-off-by: wHo-EM-i <ehteshammalik18998@gmail.com>
---
 cc/tidy.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/tidy.go b/cc/tidy.go
index bf273e98..5c6b0adf 100644
--- a/cc/tidy.go
+++ b/cc/tidy.go
@@ -88,7 +88,7 @@ func (tidy *tidyFeature) flags(ctx ModuleContext, flags Flags) Flags {
 	// If not explicitly disabled, set flags.Tidy to generate .tidy rules.
 	// Note that libraries and binaries will depend on .tidy files ONLY if
 	// the global WITH_TIDY or module 'tidy' property is true.
-	flags.Tidy = true
+	flags.Tidy = false
 
 	// If explicitly enabled, by global WITH_TIDY or local tidy:true property,
 	// set flags.NeedTidyFiles to make this module depend on .tidy files.
